.github/workflows/ci.yml

```diff
@@ -0,0 +1,47 @@
+name: CI
+on:
+- push
+- pull_request
+jobs:
+  activate:
+    runs-on: ubuntu-latest
+    if: "! endsWith(github.event.head_commit.message, '[skip ci]')"
+    steps:
+    - run: echo ok go
+  build:
+    needs: activate
+    strategy:
+      matrix:
+        os:
+        - windows-latest
+        ruby:
+        #- jruby-9.2-13.0
+        #- jruby-9.1.17.0
+        - '2.7'
+        - '2.6'
+        - '2.5'
+        - '2.4'
+        - '2.3'
+    runs-on: ${{ matrix.os }}
+    env:
+      PYGMENTS_VERSION: '~> 1.2.0'
+      SOURCE_DATE_EPOCH: '1521504000'
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 5
+    - name: Install Ruby
+      uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: ${{ matrix.ruby }}
+    - name: Configure Bundler
+      run: |
+        bundle config --local path .bundle/gems
+        bundle config --local without docs lint coverage
+    - name: Install dependencies
+      run: |
+        bundle --jobs 3 --retry 3
+    - name: Test
+      run: |
+        bundle exec ruby -w $(bundle exec ruby -e 'print File.join Gem.bindir, %q(rake)') test:all
```

Comment:
I forgot again, why you're doing this dance instead of just `bundle exec rake test:all`?

Reply:
So that I can enable `-w` which Rake doesn't provide.

Reply:
`bundle --verbose` is doing something different?

Reply:
I'm not sure what you are asking. I need Ruby warnings turned on, and before anything is loaded. There are various ways to do it (some that toggle the switch to late). This is the most reliable.

(I don't understand why rake doesn't support -w, but that is not a battle I feel like fighting).

Reply:
I think using RUBY_OPT works too.

Reply:
Nope, RUBY_OPT doesn't work in this case.

Reply:
Okay, forget it then.

Reply:
Exactly. I tried for hours to try to enable script warnings when calling rake any other way. This is the only way that works.